Docs/user_guide/about.md

```diff
@@ -22,6 +22,24 @@ The 3D Slicer software is distributed under a BSD-style open source license that
 
 To use Slicer, please read the [3D Slicer Software License Agreement](https://github.com/Slicer/Slicer/blob/master/License.txt) before downloading any binary releases of the Slicer.
 
+### Licensing Context
+
+The Slicer License was drafted in 2005 by lawyers working for Brigham and Women's Hospital (BWH), a teaching affiliate of Harvard Medical School, to be BSD-like but with a few extra provisions related to medical software.  It is specific to BWH so it's not directly reusable, but it could serve as a template for projects with similar goals.
+
+It was written in part because BWH was the prime contractor on an NIH-funded development consortium (na-mic.org) and wanted all code contributions to be compatible with ultimate use in real-world medical products (that is, commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software).  Compliance with the Slicer License was required for subcontractors, a group that included General Electric and several universities (MIT, UNC...) who all reviewed and accepted this license.
+
+The license has been in continuous use since 2005 on software (slicer.org) that has been downloaded something like a million times and has been referenced in about 12,000 academic publications (https://www.slicer.org/wiki/Main_Page/SlicerCommunity).  Some of the code is also now being used in several medical products, with this license having been reviewed and accepted by the companies involved.
+
+Here are some of the key points that BWH included to increase their comfort at distributing open source medical software.
+
+* contributors affirm that they have de-identified any patient data they contribute
+
+* contributors explicitly grant royalty free rights if they contribute code covered by a patent they control (i.e. to avoid submarine patents)
+
+* no GPL or other copyleft allowed because that would make the code basically impossible to use in real medical products
```

Comment:
What's the definition of "real medical products"? 

Does the term "basically impossible" mean that there are workarounds or is it actually "impossible"?

Reply:
"real medical products" and "basically impossible" reference the motivation above where it says "commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software"

I don't know of any GPL code being used in regulated medical devices.  The people I know who develop such products would not want to be obligated to share their code the way GPL requires so they are very careful to avoid any such code.  Code like the Linux kernel, which has [explicit exceptions](https://www.kernel.org/doc/html/latest/process/license-rules.html) to make things more flexible.

Reply:
Ok it might be helpful to organize the key points as the "What" is different and then the "Why". 

The following includes one of the "why"s for BWH decision to create this license, but really the decision was because of the 4 bullet points, not just the one mentioned in this sentence. Maybe this one was the primary reason and should be listed as the first "key" point?

> It was written in part because BWH was the prime contractor on an NIH-funded development consortium (na-mic.org) and wanted all code contributions to be compatible with ultimate use in real-world medical products (that is, commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software).

Something like:
- no GPL or other copyleft allowed. 
  - This was in part because BWH was the prime contractor on an NIH-funded development consortium (na-mic.org) and wanted all code contributions to be compatible with ultimate use in real-world medical products (that is, commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software)

Reply:
Instead of "basically impossible" it would be more accurate to say that "the risk of legal issues when using GPL licensed code in a commercial product is too high, therefore most companies prohibit their software developers from using such code".